packages/react/src/evm/components/ConnectWallet/index.tsx

```diff
@@ -63,6 +63,7 @@ interface ConnectWalletProps extends ThemeProviderProps {
     loginOptional?: boolean;
   };
   className?: string;
+  btnTitle?: string;
```

Comment:
if not using `children` this should still also allow users to pass jsx in addition to a string